### PR TITLE
SDCICD-539: Fail create if invalid aws-accounts-file provided

### PR DESCRIPTION
**Pull request by jeefy**
There was a flaw in the logic checking whether an aws-accounts-file was provided and no explicit check occurred. This explicitly checks to make sure, if aws-accounts-file is provided, it is a valid file or it fails.

### CODE REVIEW

File: cmd/osde2ectl/create/cmd.go [hunk 1]
```diff
@@ -150,20 +150,25 @@ func run(cmd *cobra.Command, argv []string) error {
 		}
 	}
 
-	if _, err := os.Stat(args.awsAccountsFile); err == nil {
-		csvfile, err := os.Open(args.awsAccountsFile)
-		if err != nil {
-			log.Fatalln("Couldn't open the csv file", err)
-		}
+	if args.awsAccountsFile != "" {
```
Comment (jharrington22) [inline cmd/osde2ectl/create/cmd.go:153]: Should we have a dedicated flag for CCS clusters? If you want CCS but forget to pass the file here we'll create OSD clusters.

**Review by jharrington22 — COMMENTED** [review]
Left a comment but otherwise the changes are good and will now catch a file if it doesn't exist.

File: cmd/osde2ectl/create/cmd.go (hunk 1 above)
Comment (jeefy) [reply cmd/osde2ectl/create/cmd.go:153]: Good idea, make it a two-key system.

File: pkg/common/providers/ocmprovider/cluster.go [hunk 2]
```diff
@@ -104,17 +104,21 @@ func (o *OCMProvider) LaunchCluster(clusterName string) (string, error) {
...
-	awsAccessKey := viper.GetString(AWSAccessKey)
-	awsSecretKey := viper.GetString(AWSSecretKey)
-
-	if awsAccount != "" && awsAccessKey != "" && awsSecretKey != "" {
-		newCluster.CCS(v1.NewCCS().Enabled(true)).AWS(
-			v1.NewAWS().
-				AccountID(awsAccount).
-				AccessKeyID(awsAccessKey).
-				SecretAccessKey(awsSecretKey))
+	if viper.GetBool(CCS) {
```
Comment (jharrington22) [inline pkg/common/providers/ocmprovider/cluster.go:107]: :+1:

**Review by jharrington22 — APPROVED** [review]
/lgtm

**Pull request merged**